packages/trader/src/App/Components/Elements/ContractAudit/__tests__/contract-audit-item.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ContractAuditItem from '../contract-audit-item';
+
+const test_id = 'test_id';
+const test_icon = 'test_icon';
+const values = {
+    value: 'test_value_1',
+    value2: 'test_value_2',
+};
+const mock_default_props = {
+    id: test_id,
+    label: 'test_label',
+    value: values.value,
+};
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    formatDate: jest.fn(() => <div>formatDate</div>),
+    formatTime: jest.fn(() => <div>formatTime</div>),
```

Comment:
```suggestion
    formatDate: jest.fn(() => '2023-10-11'),
    formatTime: jest.fn(() => '12:40:45 GMT'),
```
it would be nice to have here some values that are closer to reality since we know that formatDate returns date and  formatTime returns time :)
timestamp value would need to be changed too of course in the last test case )

Reply:
@kate-deriv btw, 'formatted date' and 'formatted time' will do as well. it's up to you) just to make it a string ;)